### PR TITLE
[Example] Combining image_classification_tflite and image_classification_onnx / Support more ONNX models

### LINKED ISSUE #319: [Example] Combining image_classification_tflite and image_classification_onnx
Please combine image_classification_tflite and image_classification_onnx together for example.

### LINKED ISSUE #345: [Example] Fill README with more ONNX models

### PR DESCRIPTION
**Pull request by Wenzhao-Xiang**
Fix #319 
Combining image_classification_tflite and image_classification_onnx as image_classification:
Move **TFliteModelImporter.js** and **OnnxModelImport.js** to **examples/util**, so they can be called more generally. And the new example DeepLab(#335 )  will use the **TFliteModelImporter.js**.

Fix #345 
Add mobilenet_v2, resnet_v1, resnet_v2, inception_v2 for onnx models.

### CODE REVIEW

**Comment by Wenzhao-Xiang** [comment]
Please help take a look! @ibelem @pinzhenx Thanks!

File: examples/image_classification/utils.js [hunk 1]
```diff
@@ -86,29 +99,34 @@ class Utils {
...
       };
-      if (progress && typeof this.progressCallback !== 'undefined')
-        request.onprogress = this.progressCallback;
-
+      if (progress) {
+        let self = this;
+        request.onprogress = function(ev) {
+          if (self.updateProgress) {
+            self.updateProgress(ev);
+        }
```
[the remark this answers is not in the document]
Comment (Wenzhao-Xiang) [reply examples/image_classification/utils.js]: Now, changed it as the following:
```js
if (progress && typeof this.updateProgress !== 'undefined') {
        request.onprogress = this.updateProgress;
      }
```
Thank you so much!

**Comment by pinzhenx** [comment]
Thanks you for merging the ONNX example. The PR LGTM.

**Comment by ibelem** [comment]
Thanks for the update! LGTM. @pinzhenx tried to optimize the UI of dropdown list but i told him i will improve the UI together for whole samples with current bootstrap framework and minimal html changes in these days.

**Comment by huningxin** [comment]
Thanks All! We finally have an all-in-one image classification demo!

**Pull request merged**